### ISSUE TITLE
In Complete Survey flow, on Temperature Screen ' 97/98 F values not allowed to enter

### ISSUE THREAD

**Issue by PallaviGanorkar** [google/android-fhir#1155]
**Describe the bug**
In Complete Survey, on Temperature Screen , select Unit as 'F' and enter Record Temperature as any normal value i. e 97 ,98, its not allowed
**Component**
 SDC library, reference app

**To Reproduce**
1. Install and launch the Demo App
2. Select one patient from list where the survey is not yet completed
3. Click on 'Complete Survey' button
4. Click on Next Button till Temperature screen
5. Here Select Unit as 'F' from drop down
6. Enter value for 97 for Record Temperature
4. See error : Record Temperature field shows error 'Maximum value allowed is:45.0'

**Expected behavior**
It should allow valid range for minimum and maximum fahrenheit value and 97 is normal human body temperature its should be allowed

**Screen
![Temperature_97F_having_erroe](https://user-images.githubusercontent.com/13957417/153197309-c4e690f8-6ace-4a32-a643-13e9f825aa0e.png)
shots**

**Smartphone (please complete the following information):**
 - Device: Nokia 3.4
 - Android version: 11
 - Build number: 00WW_2_410_SP01

**Comment by jingtang10** [comment]
I don't think this is an Android FHIR SDK issue. This is an authoring issue: the sdk cannot know that the max min values should change depending on the answer to a different question. That needs to be somehow done by the questionnaire (possibly using enable when statements to show different input questions with different max min value constraints).

under this ticket there'll be no change in the library. but we could fix the questionnarie. for example relax the value range so it makes sense for fahreheit values.
